source/game/system/map/MapdataPointInfo.cc

```diff
@@ -0,0 +1,50 @@
+#include "MapdataPointInfo.hh"
+
+#include "game/system/CourseMap.hh"
+
+namespace System {
+
+MapdataPointInfo::MapdataPointInfo(const SData *data) : m_rawData(data) {
+    u8 *unsafeData = reinterpret_cast<u8 *>(const_cast<SData *>(data));
+    EGG::RamStream stream = EGG::RamStream(unsafeData, sizeof(SData));
```

Comment:
The unsafe data pattern is antiquated now.

The stream should be initialized with `sizeof(SData) + parse<u16>(data->pointCount) * sizeof(Point)`.